src/Dialect/ONNX/ONNXShapeHelper.cpp

```diff
@@ -36,6 +36,7 @@ ONNXOpShapeHelper<OP>::ONNXOpShapeHelper(OP *newOp)
     : op(newOp), scope(nullptr, newOp->getLoc()),
       fGetDenseVal(getDenseElementAttributeFromONNXValue), fLoadVal(nullptr),
       outputsDims() {
+  assert(op && "Expecting a valid pointer");
```

Comment:
Do we need to check whether the pointer is null? A safer programming practice?

Reply:
My understanding is that `op` should never be `nullptr`.